### DISCUSSION TITLE
GraphQL IDE - Banana Cake Pop vs GraphQL Playground

### DISCUSSION THREAD

**Discussion by mahedee** [OpenCodeFoundation/eSchool#371] [Q&A]
Why do you use Banana Cake Pop instead of GraphQL Playground? What is the benefit?

**Comment by ratanparai** [comment]
Normally the GraphQL UI is not include in production build. `Banana Cake Pop` or `GraphQL Playground` is like Postman. You can use the one you like. 

For example, If you like `GraphQL Playground` more then please download it from [https://github.com/graphql/graphql-playground](https://github.com/graphql/graphql-playground) provide your service's schema address into it. 

For the GraphQL server implementation, I am using [HotChocolate](https://github.com/ChilliCream/hotchocolate) library. Banana Cake Pop is included with the library. 

There was no other special reason to choose it over other solution for now.
